### ISSUE TITLE
Error running the basic example

### ISSUE THREAD

**Issue by samlaf** [questdb/c-questdb-client#38]
Running the basic example in https://crates.io/crates/questdb-rs and getting
```
   Compiling questdb-rs v2.1.3
error: failed to run custom build command for `questdb-rs v2.1.3`

Caused by:
  process didn't exit successfully: `/Users/samlaf/devel/sui/sui-mev-bot/rust/target/debug/build/questdb-rs-b4cbfe11b6777abc/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-changed=build.rs
  cargo:rerun-if-changed=Cargo.lock
  cargo:rerun-if-changed=src/test/interop/ilp-client-interop-test.json

  --- stderr
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error("invalid type: map, expected f64", line: 27, column: 13)', /Users/samlaf/.cargo/registry/src/github.com-1ecc6299db9ec823/questdb-rs-2.1.3/build.rs:83:39
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
```

:(

**Comment by samlaf** [comment]
I narrowed it down and this happens due to some buggy interaction with another dependency of mine `sui-sdk = { git = "https://github.com/MystenLabs/sui", branch = "testnet" }`. They seem to use rocksdb, could that cause an issue?

**Comment by amunra** [comment]
This should be fix as per the latest release 3.0.0.
